### ISSUE TITLE
[Reboot] Suppress focus outline on elements that cannot be accessed via keyboard

### ISSUE THREAD

**Issue by oleersoy** [twbs/bootstrap#18330]
Hi,

[SuitCSS's base module](https://github.com/suitcss/base/blob/master/lib/base.css) contains the following snippet to suppress the focus outline on elements that cannot be accessed via keyboard.

``` css

/**
 * Suppress the focus outline on elements that cannot be accessed via keyboard.
 * This prevents an unwanted focus outline from appearing around elements that
 * might still respond to pointer events.
 */

[tabindex="-1"]:focus {
  outline: none !important;
}
```

Perhaps reboot should also include this.

Cheers,
Ole

**Comment by cvrebert** [comment]
@patrickhlauke Any concerns here?

**Comment by patrickhlauke** [comment]
no concerns i can think of just now, no.

**Comment by mdo** [comment]
Added in 5a6a128.
